lib/csv_helper/exporter.rb

```diff
@@ -3,45 +3,54 @@
 module CsvHelper
   module Exporter
     def export
+      generate(csv_headers)
```

Comment:
I don't see relevant tests for this code, am I missing something?

Reply:
I think all of the Exporter tests are through the dashboards_controller_spec